Fix eslint errors introduced with design fixes

The following errors have been introduced in our develop branch by merging #1936 ignoring the warnings of our CircleCI build system - please fix them as they might indicate bugs that could happen:

```
$ npm run lint:errors

> TreecounterApp@1.3.1 lint:errors /Users/norbertschuler/Development/PFTP/treecounter-app
> eslint app --quiet


/Users/norbertschuler/Development/PFTP/treecounter-app/app/components/PlantProjects/PlantProjectSnippet.native.js
   16:3   error  'survival_grey' is defined but never used              no-unused-vars
  118:11  error  'survivalRateLeaf' is assigned a value but never used  no-unused-vars

/Users/norbertschuler/Development/PFTP/treecounter-app/app/components/PlantProjects/PlantProjectSnippetDetails.native.js
   14:3   error  'survival_grey' is defined but never used              no-unused-vars
  120:11  error  'survivalRateLeaf' is assigned a value but never used  no-unused-vars
  126:11  error  'survivalColor' is assigned a value but never used     no-unused-vars

/Users/norbertschuler/Development/PFTP/treecounter-app/app/components/PublicTreeCounter/PublicTreecounter.native.js
  176:18  error  'Platform' is not defined  no-undef

/Users/norbertschuler/Development/PFTP/treecounter-app/app/components/TreecounterGraphics/Trillion.native.js
   7:3   error  'Image' is defined but never used          no-unused-vars
  30:10  error  'getLocalRoute' is defined but never used  no-unused-vars
  37:10  error  'trees' is defined but never used          no-unused-vars

✖ 9 problems (9 errors, 0 warnings)

```

--- a/app/components/PlantProjects/PlantProjectSnippet.native.js
+++ b/app/components/PlantProjects/PlantProjectSnippet.native.js
@@ -13,10 +13,10 @@ import { getImageUrl } from '../../actions/apiRouting';
 import {
   // tick,
   location_grey,
-  survival_grey,
-  tax_grey,
-  leaf,
-  leafGray
+  // survival_grey,
+  tax_grey
+  // leaf,
+  // leafGray
 } from '../../assets';
 import i18n from '../../locales/i18n';
 import styles from '../../styles/selectplantproject/selectplantproject-snippet.native';
@@ -66,8 +66,8 @@ class PlantProjectSnippet extends PureComponent {
       allowDonations,
       image,
       reviewScore: plantProjectRating,
-      reviews,
-      survivalRateStatus
+      reviews
+      // survivalRateStatus
       // description,
       // homepageUrl: homepageUrl,
       // homepageCaption: homepageCaption,
@@ -115,12 +115,12 @@ class PlantProjectSnippet extends PureComponent {
         );
       }
     deducibleText1 = deducibleText1.join(', ') + '.';
-    const survivalRateLeaf =
-      survivalRateStatus == 'verified'
-        ? leaf
-        : survivalRateStatus == 'self-reported'
-          ? leafGray
-          : null;
+    // const survivalRateLeaf =
+    //   survivalRateStatus == 'verified'
+    //     ? leaf
+    //     : survivalRateStatus == 'self-reported'
+    //       ? leafGray
+    //       : null;
     let onPressHandler = this.props.clickable ? this.containerPress : undefined;
 
     return (
@@ -226,8 +226,8 @@ class PlantProjectSnippet extends PureComponent {
                       marginRight: 10
                     }}
                   />
-                  
-                  
+
+
                   <View style={styles.survivalText}>
                     <View style={{ flexDirection: 'row' }}>
                       <Text style={styles.survivalText}>

--- a/app/components/PlantProjects/PlantProjectSnippetDetails.native.js
+++ b/app/components/PlantProjects/PlantProjectSnippetDetails.native.js
@@ -11,10 +11,10 @@ import {
 import { getImageUrl } from '../../actions/apiRouting';
 import {
   location_grey,
-  survival_grey,
-  tax_grey,
-  leaf,
-  leafGray
+  // survival_grey,
+  tax_grey
+  // leaf,
+  // leafGray
 } from '../../assets';
 import i18n from '../../locales/i18n';
 import styles from '../../styles/selectplantproject/selectplantproject-snippetDetails.native';
@@ -75,8 +75,8 @@ class PlantProjectSnippetDetails extends PureComponent {
       imageFile,
       image,
       reviewScore: plantProjectRating,
-      reviews,
-      survivalRateStatus
+      reviews
+      // survivalRateStatus
       // description,
       // homepageUrl: homepageUrl,
       // homepageCaption: homepageCaption,
@@ -117,13 +117,13 @@ class PlantProjectSnippetDetails extends PureComponent {
       treeCost,
       taxDeductibleCountries
     };
-    const survivalRateLeaf =
-      survivalRateStatus == 'verified'
-        ? leaf
-        : survivalRateStatus == 'self-reported'
-          ? leafGray
-          : null;
-    const survivalColor = survivalRateStatus == 'verified' ? '#89b53a' : 'gray';
+    // const survivalRateLeaf =
+    //   survivalRateStatus == 'verified'
+    //     ? leaf
+    //     : survivalRateStatus == 'self-reported'
+    //       ? leafGray
+    //       : null;
+    // const survivalColor = survivalRateStatus == 'verified' ? '#89b53a' : 'gray';
     let onPressHandler = this.props.clickable ? this.containerPress : undefined;
     return (
       <TouchableHighlight underlayColor={'white'} onPress={onPressHandler}>

--- a/app/components/PublicTreeCounter/PublicTreecounter.native.js
+++ b/app/components/PublicTreeCounter/PublicTreecounter.native.js
@@ -2,6 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
+  Platform,
   ScrollView,
   View,
   Text,

--- a/app/components/TreecounterGraphics/Trillion.native.js
+++ b/app/components/TreecounterGraphics/Trillion.native.js
@@ -4,7 +4,7 @@ import {
   ScrollView,
   Text,
   View,
-  Image,
+  // Image,
   TouchableOpacity,
   // SafeAreaView,
   Animated,
@@ -27,14 +27,14 @@ import i18n from '../../locales/i18n';
 import { updateStaticRoute } from '../../helpers/routerHelper';
 // import Leaderboard from '../../containers/Leaderboard';
 import Leaderboard from '../LeaderboardRefresh/LeaderBoard/leaderboard';
-import { getLocalRoute } from '../../actions/apiRouting';
+// import { getLocalRoute } from '../../actions/apiRouting';
 import {
   fetchpledgeEventsAction,
   fetchPublicPledgesAction
 } from '../../actions/pledgeEventsAction';
 import { loadUserProfile } from '../../actions/loadUserProfileAction';
 import { currentUserProfileSelector } from '../../selectors';
-import { trees } from '../../assets';
+// import { trees } from '../../assets';
 import tabStyles from '../../styles/common/tabbar';
 import { saveItem, fetchItem } from '../../stores/localStorage.native';
 import Constants from '../../utils/const';

